Robus/src/reception.c

```diff
@@ -134,8 +134,7 @@ void Recep_GetData(volatile uint8_t *data)
     }
     else if (data_count > data_size)
     {
-        uint16_t crc = ((uint16_t)current_msg->data[data_size]) |
-                       ((uint16_t)current_msg->data[data_size + 1] << 8);
+        uint16_t crc = ((uint16_t)current_msg->data[data_size]) | ((uint16_t)current_msg->data[data_size + 1] << 8);
```

Comment:
Replace | by ||.
Perhaps we should put OR operator || in the second line.  So, after clang format processing, the conditions should be on different lines.

Reply:
Here is a logical or so we have to keep the |